src/osp/Active/SysDebugRender.cpp

```diff
@@ -46,30 +47,51 @@ SysDebugRender::SysDebugRender(ActiveScene &rScene) :
                           std::bind(&SysDebugRender::draw, this, _1))
 {
     Package& glResources = m_scene.get_context_resources();
-    glResources.add<adera::shader::Phong>("phong_shader",
-        adera::shader::Phong{Magnum::Shaders::Phong::Flag::DiffuseTexture});
+
+    using namespace adera::shader;
+
+    glResources.add<Phong>("phong_shader",
+        Phong{Magnum::Shaders::Phong::Flag::DiffuseTexture});
+
+    glResources.add<PlumeShader>("plume_shader");
```

Comment:
Until we find a better place to load shaders, I'm going to continue stashing them in the SysDebugRender constructor, but we should find a new place soon